build.gradle

```diff
@@ -34,7 +34,7 @@ dependencies {
     minecraft "com.mojang:minecraft:${project.minecraft_version}"
     mappings "net.fabricmc:yarn:${project.yarn_mappings}:v2"
     modCompile "net.fabricmc:fabric-loader:${project.loader_version}"
-    modCompile "net.coderbot.iris:Iris:0.1.0"
+    modCompile "net.coderbot.iris:iris:0.1.0-sodium_compatibility"
```

Comment:
I still don't understand why it shows up like this, even though the one in green is exactly what is looks like in the repo, meaning no change should be done. Are you sure you forked the latest commit on the repo, because this seems weird. For context, this is what it looks like in the repo:
![image](https://user-images.githubusercontent.com/68545280/109383695-f8138100-78df-11eb-84d1-c616eb894793.png)

Reply:
does it matter, if it achieves the same result?